.buildkite/pipeline.yml

```diff
@@ -0,0 +1,86 @@
+
+steps:
+- label: ":ruby: Rails Current"
+  timeout_in_minutes: 5
+  agents:
+    queue: ruby
+    ruby: 2.6.5
+  env:
+    JUNIT_OUTPUT_DIR: test/reports/current
+  commands:
+  - apt-get update && apt-get install -y libsqlite3-dev
```

Comment:
We don't have a `ruby-sqlite3` queue. Is this something we should add for similar migrations or is manually installing sufficient?

Reply:
I think this manual install hack is enough to start with at least. I don’t think we have a lot of SQLite builds and it’s a local file system database so there’s no peripherals necessary. I think if we were to do anything, we would just install SQLite in the ruby agent rather than make a separate queue